debezium-quarkus-outbox/deployment/src/main/java/io/debezium/outbox/quarkus/deployment/DebeziumOutboxConfig.java

```diff
@@ -70,6 +70,12 @@
     @ConfigItem
     public DebeziumOutboxConfigTracingSpan tracingSpan;
 
+    /**
+     * Whether or not smallrye-opentracing extension is present.
+     */
+    @ConfigItem(name = "tracing.enabled", defaultValue = "false")
```

Comment:
This should default to true, as discussed elsewhere. Also the comment isn't quite right.